backend/de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/modelvalidator/C_OrderLine.java

```diff
@@ -39,6 +40,13 @@
 @Component
 public class C_OrderLine
 {
+
+	@Init
+	public void init()
+	{
+		CopyRecordFactory.enableForTableName(I_C_OrderLine.Table_Name);
```

Comment:
with respect to
> OBS: Enabled copyrecordfactory for salesorderline to be able to test it on at least a tab. (In the future this can be enabled for more).

Why wait for the future? Suggestion: 
* find the usages of `CopyRecordFactory.enableForTableName`.
* if the factory is enable for a parent (e.g. `C_Order` or `DD_Order`) then also add a second line *there* to enable it also for the line